### PR TITLE
Now LiAuthmanger is accessed as a propery of LiSDKManger

### PR DESCRIPTION
**Pull request by shekhardahore**
Also removed LiColor as it was redundant.

### CODE REVIEW

File: LiSDKUIComponents/HomeView/LiHomeViewController.swift [hunk 1]
```diff
@@ -151,13 +151,13 @@ open class LiHomeViewController: UIViewController {
...
-        if LiSDKManager.sharedInstance.isUserLoggedIn() {
+        if LiSDKManager.sharedInstance.liAuthManager.isUserLoggedIn() {
             let vc = LiNewMessageViewController(model: LiNewPost(boardId: nil, boardName: nil))
             let navController = UINavigationController(rootViewController: vc)
             self.present(navController, animated: true, completion: nil)
         } else {
-            LiSDKManager.sharedInstance.liLoginDelegate = self
-            LiSDKManager.sharedInstance.initLoginFlow(from: self, deviceToken: deviceToken, notificationProvider: notificationProvider)
+            LiSDKManager.sharedInstance.liAuthManager.liLoginDelegate = self
+            LiSDKManager.sharedInstance.liAuthManager.initLoginFlow(from: self, deviceToken: deviceToken, notificationProvider: notificationProvider)
```
Comment (AatishMolasi) [inline LiSDKUIComponents/HomeView/LiHomeViewController.swift]: is this self some sort of delegate?

Comment (AatishMolasi) [reply LiSDKUIComponents/HomeView/LiHomeViewController.swift]: also try and use self while accessing instance variables (makes a difference when/if you decide to use setters/getters)

Comment (shekhardahore) [reply LiSDKUIComponents/HomeView/LiHomeViewController.swift]: Yes, it's a delegate.
It's a singleton for now, but I'll keep this in mind

Comment (AatishMolasi) [reply LiSDKUIComponents/HomeView/LiHomeViewController.swift]: try and use `delegate:` instead of `from:`
Becomes more clear

**Pull request merged**